README.md

```diff
@@ -31,72 +31,582 @@ _Kev_ reduces the need for Kubernetes expertise in the team. The generated Kuber
+├── docker-compose.kev.stage.yaml       # stage Compose environment override file
+├── docker-compose.kev.prod.yaml        # prod Compose environment override file
+├── kev.yaml                            # kev project manifest
 ├── ...
 ```
 
-Your project is now ready and environment specific configuration files generated. _Kev_ will infer useful configuration details already present in the source compose file(s). If this is not possible it'll assume sensible defaults. Users have an easy way to manually alter values directly in the environment specific override file as and when required.
+Here's what happened, _Kev_,
+- Inferred the configuration details already present in your compose Kubernetes deployment sources.
+- Assigned sensible defaults for any config it couldn't infer.
+- Created Compose overrides files for the `stage` and `prod` environments.
+
+That's it, _Kev_ is now bootstrapped and ready!
 
-### Render K8s manifests
+From now on it can,
+- Detect edits in your source compose file.
+- Apply any related config changes to your compose environment overrides.
+- Generate deployment manifests.   
 
-In order to deploy your project application to Kubernetes you will need to supply _something_ K8s can understand. Currently, _Kev_ only supports Docker Compose conversion to native Kuberentes manifests. The Community might add additional output formats at later stages. See our [Roadmap][roadmap] for planned new features.
+You can now customise your deployment targets by altering values in the relevant Compose environment override file. 
 
-To render manifests, simply run the following command from your project root:
+### render
+
+We now need to generate manifests based on your Docker Compose config and environments. You'll use these manifests to deploy your app to Kubernetes. 
+
+Run the following command from your project root:
 
 ```sh
 kev render
 ```
 
-`render` generates the project's manifests based on the tracked docker-compose files, using the desired output format specified via the `-f` flag (defaults to `kubernetes`), and for selected environments. Note that ALL environments will be rendered by default if none are specified.
+The command,
+- Detects edits you made to the project's source compose file(s) to re-infer config changes.
+- Applies any found config changes to your compose environment overrides.
+- Generates kubernetes manifests based on all compose files including environment overrides.
+- Targets all environments.
 
-You can control which environments to render the manifests for with `-e` flag(s).
+The directory below should now appear in your project directory:
 
-You may also specify the output directory for generated manifests with `-d` flag. Note, that specified directory will contain sub-directories, each for a separate environment name for which manifests were generated.
+```sh
+├── k8s     # stores the Kubernetes manifests for all target deployment environments. 
+├── ...
```

Comment:
^ would be good to give examples including subdirectories for stages from the example above.

Reply:
done